packages/gatsby/src/schema/infer/__tests__/infer.js

```diff
@@ -525,7 +525,12 @@ describe(`GraphQL type inference`, () => {
           name: `Repro`,
           interfaces: [`Node`],
           fields: {
-            field_that_needs_to_be_sanitized_: `String`,
+            field_that_needs_to_be_sanitized_: {
+              type: `String`,
+              extensions: {
+                proxy: { from: `field_that_needs_to_be_sanitized?` },
+              },
```

Comment:
🤔  This is expected that `@proxy` (or resolver) is needed for those now, correct? because it's not adding "defaultResolver" anymore?

Reply:
Yes but only if you define the field via schema customization. Inferred fields still don't need this.